src/Element/WithContext/Embed.elm

```diff
@@ -0,0 +1,103 @@
+module Element.WithContext.Embed exposing
```

Comment:
Not sure if it's better to call it `Embed` or `Interop`? Probably `Embed` is better